Makefile

```diff
@@ -13,7 +13,7 @@ CLUSTERS_ARGS = --cluster_settings $(DAPPER_SOURCE)/scripts/kind-e2e/cluster_set
 clusters: build images
 
 e2e: # internally depends on deploy target, which will execute only if not already deployed
-	./scripts/kind-e2e/e2e.sh --deploytool $(deploytool)
+	./scripts/kind-e2e/e2e.sh
```

Comment:
Don't we need to pass the flags down ? how's it working now that we are calling deploy from the kind e2e? :?

Reply:
Hmm, I see it was working for globalnet that we don't pass, how? :?

Reply:
it's magic of make :)
it's passed since it's still declared in .travis.yaml

Reply:
But .. hmmm I see this:   - env: CMD="make e2e deploytool=helm globalnet=true"

How are those passed down anymore if we remove the flags when calling e2e.sh ?

Reply:
Looking at CI it's clearly working, so I'll keep the questions offline ;)